donate/templates/fragments/donate_form.html

```diff
@@ -142,6 +142,9 @@ <h2 class="donate-form__actions-heading heading heading--secondary">{% trans "Ch
         {% render_form_field braintree_form.frequency %}
         {% render_form_field braintree_form.currency %}
         {% render_form_field braintree_form.source_page_id %}
+        {% render_form_field braintree_form.landing_url %}
+        {% render_form_field braintree_form.project %}
+        {% render_form_field braintree_form.campaign_id %}
```

Comment:
@solarissmoke Would it be much more work to add/use a `render_hidden_form_fields` tag that output a hidden input for everything in `form.hidden_fields`?

Reply:
No, it wouldn't, and this has the additional benefit that the field would get rendered without the useless template wrapper that it currently gets. I'll amend to do this, thanks!